docs/griptape-tools/official-tools/google-cal.md

```diff
@@ -0,0 +1,47 @@
+# GoogleCalendarClient
+
+The GoogleCalendarClient tool allows you to interact with Google Calendar.
+
+```python
+from griptape.tools import GoogleCalendarClient
+from griptape.structures import Agent
+
+# Create the GoogleCalendarClient tool
+google_calendar_tool = GoogleCalendarClient(
```

Comment:
Example must run in integration tests. Replace dummy values like `"YOUR_PROJECT_ID"` with environment variables via `os.getenv("GOOGLE_PROJECT_ID")`